src/modules/rating/controllers/rating.controller.ts

```diff
@@ -210,4 +215,69 @@ export class RatingController {
 			throw new Error('Failed to delete rating')
 		}
 	}
+
+	@Get('/me')
+	@ApiResponse({
+		status: 200,
+		type: MyRatingResponseDTO,
+	})
+	@ApiQuery({
+		name: 'cursor',
+		type: String,
+		required: false,
+	})
+	async getMyRating(
+		@Query('productId') product_id: string,
+		@Query('user_id') user_id: string,
```

Comment:
ID of user should be retrieved via access token passed to the request. Not from @Query